core/client-internal/src/main/java/tachyon/client/block/TachyonBlockStore.java

```diff
@@ -96,11 +95,8 @@ public BlockInfo getInfo(long blockId) throws IOException {
     BlockMasterClient masterClient = mContext.acquireMasterClient();
     try {
       for (WorkerInfo workerInfo : masterClient.getWorkerInfoList()) {
-        NetAddress address = new NetAddress(workerInfo.getAddress().getHost(),
-            workerInfo.getAddress().getRpcPort(), workerInfo.getAddress().getDataPort(),
-            workerInfo.getAddress().getWebPort());
-        infoList.add(
-            new BlockWorkerInfo(address, workerInfo.getCapacityBytes(), workerInfo.getUsedBytes()));
+        infoList.add(new BlockWorkerInfo(workerInfo.getAddress(), workerInfo.getCapacityBytes(),
```

Comment:
yeah, I decided to factor those changes into a separate PR: https://github.com/amplab/tachyon/pull/2652

that PR introduces the wire types for all Thrift endpoints to be used by Tachyon clients ... another PR is needed to do the same for Thrift endpoints to be used by Tachyon workers ... I would like to cleanup the Thrift worker endpoints structures too because they seem unnecessarily complicated and inconsistent